packages/itwin/grouping-mapping-widget/src/widget/components/Grouping.tsx

```diff
@@ -459,11 +458,11 @@ export const Groupings = ({ mapping, goBack }: GroupsTreeProps) => {
                                 styleType="borderless"
                                 className="gmw-group-view-icon"
                               >
-                                <SvgRectangle
-                                  stroke-width="100"
-                                  stroke={getGroupColor(
-                                    groups.findIndex((group) => g.id === group.id),
-                                  )}
+                                <div
+                                  className="gmw-color-box"
```

Comment:
Nit pick. Can we name this `gmw-color-legend` instead?

Reply:
Done